openedx/features/course_experience/utils.py

```diff
@@ -149,8 +149,10 @@ def recurse_num_graded_problems(block):
         """
         is_scored = block.get('has_score') and block.get('weight', 1) > 0
         is_graded = block.get('graded')
+        is_countable = block.get('type') not in ('lti', 'lti_consumer')
```

Comment:
Without this bit, LTI units would show up as 1 problem. That doesn't seem correct, so we just exclude LTIs from the count. They'll still show a graded icon.